erpnext/selling/doctype/customer/customer.py

```diff
@@ -352,20 +352,20 @@ def get_customer_list(doctype, txt, searchfield, start, page_len, filters=None):
 		match_conditions += "{}".format(filter_conditions)
 
 	return frappe.db.sql("""
-			select %s
-			from `tabCustomer`
-			where docstatus < 2
-				and (%s like %s or customer_name like %s)
-					{match_conditions}
-			order by
-				case when name like %s then 0 else 1 end,
-				case when customer_name like %s then 0 else 1 end,
-				name, customer_name limit %s, %s
+		select %s
+		from `tabCustomer`
+		where docstatus < 2
+			and (%s like %s or customer_name like %s)
+			{match_conditions}
+		order by
+			case when name like %s then 0 else 1 end,
+			case when customer_name like %s then 0 else 1 end,
+			name, customer_name limit %s, %s
 		""".format(match_conditions=match_conditions) % (", ".join(fields), searchfield, "%s", "%s", "%s", "%s", "%s", "%s"),
-			("%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, start, page_len))
+		("%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, start, page_len))
 
 
-def check_credit_limit(customer, company, reference_doctype, reference_document, ignore_outstanding_sales_order=False, extra_amount=0):
+def check_credit_limit(customer, company, ignore_outstanding_sales_order=False, extra_amount=0):
```

Comment:
Please put in a detailed docstring explaining what this function does. Ensure you mention the arguments passed into it and what they represent (if any) and the values returned by it (if any).

Reply:
> @vishdha Not all the changes have been pulled in. If its an issue, just copy and paste it in your instance and raise the PR

there 6 commit upstream cherry-picked all of them. I dont think there is an extra commit.